Code/Framework/AzToolsFramework/AzToolsFramework/UI/DocumentPropertyEditor/DPEComponentAdapter.h

```diff
@@ -59,8 +59,14 @@ namespace AZ::DocumentPropertyEditor
 
         void OnEntityDestroyed(const AZ::EntityId&) override;
 
+    private:
+        //! Checks if the component is still valid in the entity.
+        bool IsComponentValid();
```

Comment:
Nit: Can make this const
```suggestion
        bool IsComponentValid() const;
```